content/components/banner.mdx

```diff
@@ -8,15 +8,254 @@ tags:
   - flash
 ---
 
+import {Box} from '@primer/react'
 import ComponentLayout from '~/src/layouts/component-layout'
 export default ComponentLayout
 import {AccessibilityLink} from '~/src/components/accessibility-link'
 
-Banner can be used in one of two ways:
+<img
+  width="960"
+  alt="Image displaying each variant of banenr, from top: critical, warning, success, info, upsell."
```

Comment:
This is quite nitpicky, but the specific text you've used in all the images is for a message I think we would see in a global Announcement above the navbar. Is there something more generic you could use to not cause confusion?

Reply:
Yeah, I thought about this as well. Do you have good examples from GitHub in mind, or should we make up random ones?

Reply:
I think what you came up with looks perfect! 🎉

Reply:
Only thing I'm noticing is this image feels really tiny and hard to read. What do you think?

<img width="1005" alt="header image for banner" src="https://github.com/primer/design/assets/18661030/492a6b87-9bd1-4a4f-be3d-8b6622e8c430">

Reply:
Yeah, true, I'll resize it.